pkgs/tools/misc/rot8/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rot8";
+  version = "0.1.5";
+
+  src = fetchFromGitHub {
+    owner = "efernau";
+    repo = pname;
+    rev = "7ed926f6d94a999cd2cb702af9f3de0f566095b1";
```

Comment:
please use release tag rather than revision

Reply:
I tried using the tag initially, but it fails to build in that case, because https://github.com/efernau/rot8/archive/v0.1.5.tar.gz returns: `the given path has multiple possibilities: #<Git::Ref:0x00007fbc93169890>, #<Git::Ref:0x00007fbc931696b0>`.

Reply:
this should work

```suggestion
    rev = "refs/tags/v${version}";
```

Reply:
Ah, good thinking! I added your suggestion. Appreciate you taking the time to figure that out, I can see how it will make future updates a bit easier.